linebot/models/base.py

```diff
@@ -78,21 +78,21 @@ def as_json_dict(self):
         :return: dict
         """
         data = {}
-        for key in self.__dict__.keys():
+        for key, sub_obj in self.__dict__.items():
```

Comment:
Thanks ! It became cool.
However, it seems that obj and sub_obj are opposite.

How about changing to `value`, `item`?

```python
        for key, value in self.__dict__.items():
            camel_key = utils.to_camel_case(key)
            if isinstance(value, (list, tuple, set)):
                data[camel_key] = list()
                for item in value:
                    if hasattr(item, 'as_json_dict'):
                        data[camel_key].append(item.as_json_dict())
                    else:
                        data[camel_key].append(item)

            elif hasattr(value, 'as_json_dict'):
                data[camel_key] = value.as_json_dict()

            else:
                data[camel_key] = value

```

Reply:
That's better !